### PR TITLE
WIP: Add asset issue function

### PR DESCRIPTION
**Pull request by zhangyelong**
Resolves: #1472

### CODE REVIEW

File: client/asset/cli/flags.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	FlagFamily    		= "family"
+	FlagName	  		= "name"
+	FlagSymbol	  		= "symbol"
+	FlagSource	  		= "source"
+	FlagInitialSupply	= "initial-supply"
+	FlagMaxSupply 		= "max-supply"
+	FlagDecimal			= "decimal"
+	FlagMintable		= "mintable"
+	FlagOperators 		= "operators"
+)
```
Comment (chengwenxi) [inline client/asset/cli/flags.go]: go fmt

File: client/asset/cli/sendTx.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+		Example: "iriscli asset issue iris --family=00 --name=IRISnet --symbol=iris --source=00 --initial-supply=100000 --max-supply=1000000000000 --decimal=0 --mintalbe=false  --operators=<account address A>,<account address B>",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc).WithLogger(os.Stdout).
+				WithAccountDecoder(utils.GetAccountDecoder(cdc))
+			txCtx := utils.NewTxContextFromCLI().WithCodec(cdc).
+				WithCliCtx(cliCtx)
+
+			//fromStr of the issuer
+			fromStr, err := cliCtx.GetFromAddress()
+			_=fromStr
```
Comment (chengwenxi) [inline client/asset/cli/sendTx.go]: remove debug code

File: client/asset/cli/sendTx.go [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+			if err != nil {
+				return err
+			}
+
+			//get the familyStr of asset
+			familyStr:=viper.GetString(FlagFamily)
+			family,err:= hex.DecodeString(familyStr)
+			_=family
```
Comment (chengwenxi) [inline client/asset/cli/sendTx.go]: remove debug code

**Comment by haifengxi** [comment]
Replaced by: #1532

**Pull request closed without merging**